worker/revpi_factory_reset.sh

```diff
@@ -27,6 +27,8 @@ ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDTBh7TFMbVgVDALMMg4VWRO/lnlxy0h4SLlmBxo11PR
 EOX
 
 sshpass -p "$DEFAULT_PASS" ssh-copy-id -i /tmp/lava-key.pub "$DEFAULT_USER"@"$SSH_HOST_RPI"
+ssh pi@revpi32346 "sudo cp -r /home/pi/.ssh /root"
+ssh pi@revpi32346 "sudo chown -R root: /root/.ssh"&& rm /tmp/lava-key.pub"
```

Comment:
```suggestion
sshpass -p "$DEFAULT_PASS" ssh -o StrictHostKeyChecking=no "$DEFAULT_USER"@"$SSH_HOST_RPI" "sudo chown -R root: /root/.ssh"
```

What is the `rm /tmp/lava-key.pub` for? If your intention was to remove the tmp file we created in #4, then please add another commit for this. Also the removal must be run on the worker (eg. after the `ssh-copy-id` line).

Reply:
We do not need another commit...